src/ngraph_encapsulate_op.cc

```diff
@@ -486,9 +486,9 @@ class NGraphEncapsulateOp : public OpKernel {
       if (s_ng_backend_name != "CPU") {
         for (size_t i = 0; i < output_caches.size(); ++i) {
           void* dst_ptr;
-          std::shared_ptr<ng::runtime::TensorView> dst_tv;
+          std::shared_ptr<ng::runtime::Tensor> dst_tv;
           std::tie(dst_ptr, dst_tv) = output_caches[i];
-          auto ng_element_type = dst_tv->get_tensor().get_element_type();
+          auto ng_element_type = dst_tv->get_element_type();
           dst_tv->read(dst_ptr, 0,
```

Comment:
@avijit-nervana Removed my PR that was fixing this, since this change contains all the changes.